index.html

```diff
@@ -0,0 +1,119 @@
+            <p class="top_section_text_title">
+              일상의 모든 물건을<br />거래해보세요
+            </p>
+            <a class="top_section_text_button" href="items.html"
+              >구경하러 가기</a
+            >
+          </div>
+          <div class="top_section_image">
+            <img src="img/Img_home_top.png" alt="메인 이미지" />
+          </div>
+        </div>
+      </section>
+
+      <div class="container">
+        <section class="item">
+          <div class="background_box">
+            <div class="item_image">
+              <img src="img/Img_home_01.png" alt="item1" />
+            </div>
+            <div class="item_text">
+              <p class="item_text_type">Hot item</p>
+              <p class="item_text_title">인기 상품을<br />확인해 보세요</p>
+              <p class="item_text_content">
+                가장 HOT한 중고거래 물품을<br />판다 마켓에서 확인해 보세요
+              </p>
+            </div>
+          </div>
+        </section>
+        <section class="item">
+          <div class="background_box_right">
+            <div class="item_text">
+              <p class="item_text_type">Search</p>
+              <p class="item_text_title">
+                구매를 원하는<br />상품을 검색하세요
+              </p>
+              <p class="item_text_content">
+                구매하고 싶은 물품은 검색해서<br />쉽게 찾아보세요
+              </p>
+            </div>
+            <div class="item_image">
+              <img src="img/Img_home_02.png" alt="item1" />
+            </div>
+          </div>
+        </section>
+        <section class="item">
+          <div class="background_box">
+            <div class="item_image">
+              <img src="img/Img_home_03.png" alt="item1" />
+            </div>
+            <div class="item_text">
+              <p class="item_text_type">Regitster</p>
+              <p class="item_text_title">
+                판매를 원하는<br />상품을 등록하세요
+              </p>
+              <p class="item_text_content">
+                어떤 물건이든 판매하고 싶은 상품을<br />쉽게 등록하세요
+              </p>
+            </div>
+          </div>
+        </section>
+      </div>
+
+      <section class="bottom_section_container">
+        <div class="bottom_section_container_all">
+          <div class="bottom_section_text">
+            <p class="bottom_section_text_title">
+              믿을 수 있는<br />판다마켓 중고 거래
+            </p>
+          </div>
+          <div class="bottom_section_image">
+            <img src="img/Img_home_bottom.png" alt="메인 이미지" />
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer_left">
+        <p>©codeit - 2024</p>
+      </div>
+      <div class="footer_center">
+        <a class="footer_center_link" href="privacy.html"> Privacy Policy</a>
+        <a class="footer_center_link" href="faq.html">FAQ</a>
+      </div>
+      <div class="footer_right">
+        <a href="https://www.facebook.com">
+          <img src="img/ic_facebook.png" alt="facebook" />
+        </a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
      <ul class="footer_right">
        <li>
          <a href="https://www.facebook.com">
            <img src="img/ic_facebook.png" alt="facebook" />
          </a>
        </li>
```